### PR TITLE
Reverting the decision to make nicer looking link targets

### PR DESCRIPTION
**Pull request by kof**
User should know the underlying platform

this has also resulted in the need to have target wrapped in all links components

Screenshot 2023-03-09 at 14 19 50

## Steps for reproduction

1. create link
2. select target prop

## Code Review

- [ ] hi @rpominov , I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Review by TrySound — APPROVED** [review]

**Review by rpominov — APPROVED** [review]
Probably not in scope of this PR, but something that came to mind:

Target can also point to an iframe instead of using a keyword like `_blank`. So maybe we should leave it as `string` instead of enum.

**Comment by kof** [comment]
@rpominov how do we express that? do we have a combobox type?

**Comment by rpominov** [comment]
> do we have a combobox type?

No, only option is just a text input currently

**Comment by kof** [comment]
> No, the only option is a Text input currently

This is not an option, because it requires user to know the standard targets, we need a combobox for this, will keep a select for now, using a custom target is something I only know about but never used in practice

File: packages/prisma-client/prisma/migrations/20230309142820_link-target/migration.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        for (const build of builds) {
+          const props = JSON.parse(build.props);
+          let hasConversion = false;
+
+          // prop.value === 'self' > prop.value === '_self'
+          for (const prop of props) {
+            if (
+              prop.name === "target" &&
+              typeof prop.value === "string" &&
+              prop.value[0] !== "_"
```
Comment (TrySound) [inline packages/prisma-client/prisma/migrations/20230309142820_link-target/migration.ts:48]: startsWith('_') a little cleaner

**Review by TrySound — APPROVED** [review]

File: packages/prisma-client/prisma/migrations/20230309142820_link-target/migration.ts (hunk 1 above)
Comment (kof) [reply packages/prisma-client/prisma/migrations/20230309142820_link-target/migration.ts:48]: agree, don't care for migration though

**Pull request merged**